curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-016.md

```diff
@@ -11,10 +11,18 @@ Now that you have the CSS in the `styles.css` file, go ahead and remove the `sty
 
 # --hints--
 
-Test 1
+You should not have any `style` tags in your code.
```

Comment:
just a general note: by removing the style element from the from the html, the css styles are still applied and the text is still centered.

Reply:
That's an editor bug, right?

Reply:
@ojeytonwilliams @ahmadabdolsaheb The original intent was if the CSS in the `styles.css` tab but the applicable `rel` element was not present in the `index.html` tab and did not link to `styles.css`, then the CSS would not be applied.  This way, it appears and functions as it would in real web development.  Is that not going to happen now?  Having the CSS magically work was something we wanted to avoid.

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/42205 should resolve this.